src/Integration/Integration.csproj

```diff
@@ -97,11 +98,7 @@
        NB the target is based on the one from Microsoft.VisualStudio.SDK.EmbedInteropTypes above. -->
   <Target Name="HackIncludeALMAuthInVsix" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
     <ItemGroup>
-      <ReferencePath Condition="
-              '%(FileName)' == 'Microsoft.Alm.Authentication'
-           or '%(FileName)' == 'Microsoft.Alm.Git'
-           or '%(FileName)' == 'Microsoft.Vsts.Authentication'
-              ">
+      <ReferencePath Condition="&#xD;&#xA;              '%(FileName)' == 'Microsoft.Alm.Authentication'&#xD;&#xA;           or '%(FileName)' == 'Microsoft.Alm.Git'&#xD;&#xA;           or '%(FileName)' == 'Microsoft.Vsts.Authentication'&#xD;&#xA;              ">
```

Comment:
VS doesn't seem to like line breaks in conditions so it keeps trying to change them to control characters.
It's probably better to remove the control characters and just have the conditions on a single (long) line.